modules/backend/assets/js/backend.js

```diff
@@ -236,3 +236,309 @@ if (!!window.MSInputMethodContext && !!document.documentMode) {
         ))
     }
 }
+
```

Comment:
@ayumihamsaki I feel like this would be better to live in a separate file that gets loaded by storm.js, thoughts?